test/regress/regress1/fmf/issue4225-univ-fun.smt2

```diff
@@ -0,0 +1,6 @@
+; COMMAND-LINE: --finite-model-find
```

Comment:
shouldn't this benchmark be using `--uf-ho`?

Reply:
Yes I'll add.